code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1058,6 +1058,26 @@ datum/reagent/medicine/syndicate_nanites/on_mob_life(mob/living/M)
 	..()
 	. = 1
 
+/datum/reagent/medicine/miningnanites
+	name = "Nanites"
+	id = "miningnanites"
+	description = "It's mining magic. We don't have to explain it."
+	color = "#C8A5DC" // rgb: 200, 165, 220
+	overdose_threshold = 3 //To prevent people stacking massive amounts of a very strong healing reagent
```

Comment:
Nanites heal ALL clone damage and status effects PER TICK.

This includes stuns/weakens; they get reset to 0 PER TICK.

This is an admin reagent. It should have never been put in. Not for 500 pts. Not for 5000 pts. Not even for 99999999999999 pts.